src/test/MeshManagerTests.cpp

```diff
@@ -25,15 +25,15 @@ namespace blitzdg {
         Describe(MeshManager_Object) {
             using find_vector_type = vector<iterator_range<string::iterator>>;
 
-            MeshManager * meshManager=nullptr;
-            string *ExePath = nullptr;
+            MeshManager* restrict meshManager=nullptr;
+            string* restrict ExePath = nullptr;
```

Comment:
Don't use a pointer to string - just use a string. Replace the test for ExePath == nullptr with ExePath.empty().